### PR TITLE
DDF-5038 Correct the Solr documentation

### PR DESCRIPTION
**Pull request by ahoffer**
## What does this PR do?
Corrects outdated documentation about Solr.

#### Who is reviewing it? 
@brjeter  @tyler30clemens @austinsteffes @ricklarsen @clockard 

#### Select relevant component teams: 
@codice/docs 

#### How should this be tested?
Proof read the changes.

Fixes: #5038

#### Screenshots
image
image

#### Checklist:
- [X] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_managing/_configuring/hardening-solr.adoc [hunk 1]
```diff
@@ -57,6 +57,13 @@ file `${home_directory}/etc/custom.system.properties`
 
 ==== Change the Password to a Specific String
 
+[NOTE]
+====
+If ${branding} has generated a encrypted password, the password must be reset before
```
Comment (tyler30clemens) [inline distribution/docs/src/main/resources/content/_managing/_configuring/hardening-solr.adoc]: has generated **_an_** encrypted password

File: distribution/docs/src/main/resources/content/_managing/_configuring/solr-admin-ui.adoc [hunk 2]
```diff
@@ -7,31 +7,18 @@
...
-
-The Solr server configuration is secure by default. Solr server requires a TLS connection
-with client authentication. Solr only allows access to clients that present
-a trusted certificate.
-
-=== Using ${branding} Keystores
-
-Solr server uses the same keystores as ${branding}. A simple way to enable access to the
-Solr Admin UI is to install ${branding}'s own private key/certificate entry into a browser. The
-method to export ${branding}'s private key/certificate entry depend on
```
Comment (tyler30clemens) [inline distribution/docs/src/main/resources/content/_managing/_configuring/solr-admin-ui.adoc]: private key/certificate entry **_depends_** on

Comment (ahoffer) [reply distribution/docs/src/main/resources/content/_managing/_configuring/solr-admin-ui.adoc]: Deleted text

**Comment by ahoffer** [comment]
build now

**Pull request merged**